app/test_main.py

```diff
@@ -1,3 +1,50 @@
+from typing import Any
+
 from app.main import get_human_age
+import pytest
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (23, 23, [1, 1]),
+        (24, 24, [2, 2]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
```

Comment:
Test against negative numbers and values that differ for `cat_age` and `dog_age` to follow checklist